Mathlib/Algebra/Group/Pi/Units.lean

```diff
@@ -0,0 +1,36 @@
+/-
+Copyright (c) 2024 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import Mathlib.Algebra.Group.Pi.Basic
+import Mathlib.Algebra.Group.Units.Defs
+import Mathlib.Algebra.Group.Equiv.Basic
+import Mathlib.Util.Delaborators
+
+/-! # Units in pi types -/
+
+variable {ι} {M : ι → Type*} [∀ i, Monoid (M i)] {x : Π i, M i}
+
+/-- The units of a product is isomorpic to the product of the units. -/
+@[to_additive (attr := simps)
+  "The (additive) units of a product is isomorpic to the product of the (additive) units."]
+def Units.piEquiv : (Π i, M i)ˣ ≃* Π i, (M i)ˣ where
```

Comment:
This is inconsistent with `MulEquiv.prodUnits`; can we bring them in sync one way or the other?

Reply:
(and similarly for the docstring)

Reply:
I think they are consistent? [MulEquiv.prodUnits](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Group/Prod.html#MulEquiv.prodUnits)

Reply:
Oh sorry, you mean the name ...